src/servers/src/otlp/trace/attributes.rs

```diff
@@ -30,9 +30,20 @@ impl<'a> From<&'a AnyValue> for OtlpAnyValue<'a> {
     }
 }
 
+impl OtlpAnyValue<'_> {
+    pub fn none() -> Self {
+        Self(&AnyValue { value: None })
+    }
+}
+
+/// specialize Display when it's only a String
 impl Display for OtlpAnyValue<'_> {
     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
-        write!(f, "{}", serde_json::to_string(self).unwrap_or_default())
+        if let Some(StringValue(v)) = &self.0.value {
+            write!(f, "{v}")
+        } else {
+            write!(f, "{}", serde_json::to_string(self).unwrap_or_default())
```

Comment:
I have been thinking about our usage of `unwrap_or_default` here. It may not be the best choice for error handling because the error will be ignored. 

In next refactor I suggest that we can get rid of `to_string` to serialization of out Otlp data, and switch to a `to_text` which returns `Result<()>`